src/components/Avatar/styled.ts

```diff
@@ -1,18 +1,13 @@
-import styled, { css } from 'styled-components'
+import styled from 'styled-components'
 import { UserCircle } from '@styled-icons/fa-solid'
 
-const border = css`
-	border: 0.2em solid ${({ theme }) => theme.text.primary};
-	border-radius: 50%;
-`
-
 export const Picture = styled.img`
```

Comment:
This looks good but sometimes I get black "border" between image and it's white border. I don't know why, maybe it's because of messages we're fetching from foreign API.
![avatar](https://user-images.githubusercontent.com/76131194/111761051-b99a3200-889f-11eb-8c83-525802941a49.png)

Reply:
I know what you mean and I have no idea how to get rid of it. I tried different ways and none of them solved this problem for me. I'll try to think again. Another option to remove the border 😅

Reply:
Maybe uploading our own avatars will solve the issue. (They might be uploading some images as a circle with border? While we are uploading rectangular images and setting them as cover.)